start.sh

```diff
@@ -43,7 +47,7 @@ main() {
     source ~/ros2_ws/install/setup.bash
 
     cd carla-client
-    pids+=($(start_process "python3 ./manual_control_steeringwheel.py --sync --rolename ego_vehicle --filter vehicle.tesla.model3 -i can0" "../$log_dir/control.log"))
+    pids+=($(start_process "python3 ./manual_control_steeringwheel.py --sync --rolename ego_vehicle --filter vehicle.tesla.model3 -i vcan0" "../$log_dir/control.log"))
```

Comment:
let's leave this can0 for now. I'll create another PR to fix this

Reply:
ok